packages/@react-spectrum/meter/stories/Meter.stories.tsx

```diff
@@ -12,14 +12,15 @@
 
 import {ComponentMeta, ComponentStoryObj} from '@storybook/react';
 import {Meter} from '../';
+import {NumberFormatOptions} from '@internationalized/number';
 import React from 'react';
 
 type MeterStory = ComponentStoryObj<typeof Meter>;
 
 const formatOptions = {
   style: 'currency',
   currency: 'JPY'
-};
+} as NumberFormatOptions;
```

Comment:
Why not

```suggestion
} satisfies NumberFormatOptions;
```

?

Reply:
that's a great suggestion, and the answer would be because I'm not as good at TS as I'd like to be

Reply:
thank you for the review, i've now learned a couple things